src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -38,7 +50,7 @@
     <None Include="project.json" />
     <None Include="windows\project.json" />
```

Comment:
I see that the windows project.json adds `System.Net.Http.WinHttpHandler` to the `dnxcore50` framework dependencies and `Microsoft.TargetingPack.Private.NetNative` to `netcore50`. I don't know what makes this necessary, though. (I also don't remember being involved in this addition--is there a specific reason you pinged me?)